create_eagle_env.sh

```diff
@@ -26,13 +26,13 @@ MY_CONDA_PREFIX="$CONDA_ENVS_DIR/$MY_CONDA_ENV_NAME"
 echo "Creating $MY_CONDA_PREFIX"
 module load conda
 conda remove -y --prefix "$MY_CONDA_PREFIX" --all
-conda create -y --prefix "$MY_CONDA_PREFIX" -c conda-forge "pyarrow>=7.0.0" "python=3.9" "numpy>=1.20.0" "pandas>=1.0.0,!=1.0.4" "dask>=2022.10.0" "distributed>=2021.5" ruby
-source deactivate 
-source activate "$MY_CONDA_PREFIX"
+mamba create -y --prefix "$MY_CONDA_PREFIX" -c conda-forge "python=3.10" "pyarrow" "numpy" "pandas" "dask>=2022.10.0" "distributed" "ruby"
+conda deactivate
+conda activate "$MY_CONDA_PREFIX"
 which pip
 if [ $DEV -eq 1 ]
 then
-    pip install --ignore-installed -e .
+    pip install --no-cache-dir --ignore-installed -e .
 else
-    pip install --ignore-installed .
+    pip install --no-cache-dir --ignore-installed .
```

Comment:
These are changes from #323. I branched from that branch for this work. They will disappear once that is merged.